src/app/components/bootstrap.tsx

```diff
@@ -27,7 +27,13 @@ const Bootsrap: React.FC = () => {
           console.error("Failed to add frequency", freq, station);
           return;
         }
-        useRadioState.getState().addRadio(freq, station);
+        useRadioState
+          .getState()
+          .addRadio(
+            freq,
+            station,
+            useSessionStore.getState().getStationCallsign()
```

Comment:
See my notes in the PR about this. I'm not a huge fan.

Reply:
I'm not a fan but we have no choice as we cannot call the sessionStore outside of a functional component, I think this is honestly fine for now